.github/workflows/merge-main.yml

```diff
@@ -192,7 +94,7 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           package: ${{ github.event.repository.name }}
           owner: orgs/bcgov
-          skip_tags: prod-frontend,prod-backend
+          skip_tags: prod-service-api
 
   deploy-test:
```

Comment:
I'm not sure if I understood your question. Yeah, we're removing all references to front-end, but we should keep production.

Reply:
Sorry, GitHub wouldn't let me comment on the pertinent lines, since they're not close to any changes. I'm referring to the following lines:

123: `oc process -f .github/openshift/deploy.frontend.yml -p ZONE=${{ env.ZONE }} \
         -p PROMOTE=${{ github.repository }}:${{ env.ZONE }}-frontend | oc apply -f -`
129: `oc rollout status dc/${{ env.NAME }}-${{ env.ZONE }}-frontend -w`
159: `- zap-frontend`
183: `oc process -f .github/openshift/deploy.frontend.yml -p ZONE=${{ env.ZONE }} \
         -p PROMOTE=${{ github.repository }}:${{ env.PREV }}-frontend | oc apply -f -`
189: `oc rollout status dc/${{ env.NAME }}-${{ env.ZONE }}-frontend -w`

And the whole *Promote Frontend Image to PROD* step from `deploy-prod`.

These reference the front-end, which has its own project now, and I thought they should be removed. But after our conversation yesterday my understanding is that the deployment is out of this PR's scope, and that we'll cover it in another task soon.

So I think we're good for now.